lib/store/index.js

```diff
@@ -34,6 +37,20 @@ class Store {
     return this.editor ? this.editor.getPath() : null;
   }
 
+  @computed
+  get notebook(): ?string {
```

Comment:
I don't know it `store` is the right place to put this functionality since it is only used in one place right now. On the other hand `mobx` evaluates lazily so we can leave it here for now.
I guess it is better to return the notebook instead of the stringified version but we can figure this out later when `store.notebook` is used in multiple places.
Feel free to just ignore this comment 😉

Reply:
I viewed the notebook as something derivable from the state, so I put it as a computed property, but I was so uncertain about this, so I made the WIP PR.

Reply:
Sure that's great with me. 👍